formats/developers/using-ome-xml.txt

```diff
@@ -378,13 +380,14 @@ Possible included objects
     - ``<xs:element ref="Filter">``
     - ``<xs:element ref="FilterSet">``
     - ``<xs:element ref="FilterSetRef">``
-    - ``<xs:element ref="Group">``
-    - ``<xs:element ref="GroupRef">``
+    - ``<xs:element ref="GenericExcitationSource">``
+    - ``<xs:element ref="HashSHA1">``
```

Comment:
OMERO-side we have some non-SHA1 hashes now, I don't know if we'll have a model to-do there someday.